airflow/contrib/executors/kubernetes_executor.py

```diff
@@ -346,6 +346,7 @@ def run_next(self, next_job):
             namespace=self.namespace, worker_uuid=self.worker_uuid,
             pod_id=self._create_pod_id(dag_id, task_id),
             dag_id=dag_id, task_id=task_id,
+            try_number=try_number,
```

Comment:
Since you're not breaking all the arguments into their own line (with redundant comma) 349 could have been appended to 348 without exceeding line length.